src/components/MultiStepIndicator/__stories__/multiStepIndicator.stories.mdx

```diff
@@ -194,6 +194,52 @@ There are 4 types: Primary, success, danger, dark.
   </Story>
 </Canvas>
 
+### Modes
+
+There are 2 modes: Regular, Compact
+
+<Canvas>
+  <Story name="Modes">
+    {() => {
+      const steps = useMemo(
+        () => [
+          {
+            key: "FULFILLED",
+            status: MultiStepIndicator.stepStatuses.FULFILLED,
+            titleText: "Fulfilled title",
+            subtitleText: "Fulfilled subtitle"
+          },
+          {
+            key: "ACTIVE",
+            status: MultiStepIndicator.stepStatuses.ACTIVE,
+            titleText: "Active title with long title",
+            subtitleText: "Active subtitle"
+          },
+          {
+            key: "PENDING",
+            status: MultiStepIndicator.stepStatuses.PENDING,
+            titleText: "Pending",
+            subtitleText: "Pending subtitle"
+          }
+        ],
+        []
+      );
+      return (
+        <div className="monday-storybook-multiStepIndicator_column-wrapper">
```

Comment:
As we discuss we have components for description and for layout which should be better for this case (will be consistent with other stories and code docs will be less confusing) - please use Flex and StoryDescription. You can see examples on Avatar or AttentionBox docs

Reply:
I know that here we use these divs but these stories are relatively old